package.json

```diff
@@ -62,7 +65,7 @@
     "eslint": "^8.10.0",
     "eslint-config-prettier": "^8.4.0",
     "eslint-plugin-jsx-a11y": "^6.5.1",
-    "eslint-plugin-react": "^7.29.2",
+    "eslint-plugin-react": "7.28.0",
```

Comment:
Why are you downgrading the already updated package??

Reply:
Already mentioned the reason [here](https://github.com/fossasia/open-event-next/pull/309#discussion_r814985920)

Also, I tried to use `"eslint-plugin-react": "^7.29.2"` , but getting this warning. We need to downgrade the TypeScript version in this case.
<be />
![Capture](https://user-images.githubusercontent.com/58071992/156411362-012ddd77-f9de-40a6-aaed-49cebc6ca811.PNG)

Reply:
Already mentioned the reason [here](https://github.com/fossasia/open-event-next/pull/309#discussion_r814985920)

nvm, updating it.